src/main/resources/pilotResources/localization/eng/PilotMod-Card-Strings.json

```diff
@@ -135,5 +139,13 @@
     "NAME": "Smoke Bomb",
     "DESCRIPTION": " Draw !M! cards. NL Fill your hand with *Dazed.",
     "UPGRADE_DESCRIPTION": "Retain. NL Draw !M! cards. NL Fill your hand with *Dazed."
+  },
+  "pilot:Rush": {
+    "NAME": "Rush",
+    "DESCRIPTION": " Discard !M! card. NL Add 1 Mastiff to your hand."
+  },
+  "pilot:Mastiff": {
+    "NAME": "Mastiff",
+    "DESCRIPTION": " Deal !D! Damage and gain !M! Momentum. NL Advantage: Double the damage dealt."
```

Comment:
The card currently doesn't double the damage, it adds a second damage instance. If you want it to double the damage, you can add a calculateBonusBaseDamage() call, in which you have something like 
`if player.hasAdvantage() {
return baseDamage; 
}`
I believe that should do the thing.

Reply:
Oh nice! Yeah I initially wanted it to double but i couldn't figure it out, so I just quickly changed it to attack twice. I noticed that later when testing after uploading this.